JPS_Ontology/ontology/ontoems/OntoEMSExtension.csv

```diff
@@ -0,0 +1,107 @@
+Range,Class,,,,,,The range of the current water level reading,https://www.theworldavatar.com/kg/ontoems/,Range
+Trend,Class,,,,,,The trend of the current water level reading,https://www.theworldavatar.com/kg/ontoems/,Trend
+NormalRange,Class,Range,IS-A,,,,Current water level is within normal range,https://www.theworldavatar.com/kg/ontoems/,Normal range
+LowRange,Class,Range,IS-A,,,,Current water level is below normal range,https://www.theworldavatar.com/kg/ontoems/,Low range
+HighRange,Class,Range,IS-A,,,,Current water level is above normal range,https://www.theworldavatar.com/kg/ontoems/,High range
+UnavailableRange,Class,Range,IS-A,,,,Current water level range cannot be determined,https://www.theworldavatar.com/kg/ontoems/,Unavailable range
+Steady,Class,Trend,IS-A,,,,Current water level has been (approximately) constant over last 12 hours,https://www.theworldavatar.com/kg/ontoems/,Steady (no trend)
+Rising,Class,Trend,IS-A,,,,Current water level has been rising over last 12 hours,https://www.theworldavatar.com/kg/ontoems/,Rising trend
+Falling,Class,Trend,IS-A,,,,Current water level has been falling over last 12 hours,https://www.theworldavatar.com/kg/ontoems/,Falling trend
+UnavailableTrend,Class,Trend,IS-A,,,,Current water level trend cannot be determined,https://www.theworldavatar.com/kg/ontoems/,Unavailable trend
+hasCurrentRange,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,Range,,Relationship between a water level measure and its current range,https://www.theworldavatar.com/kg/ontoems/,has current range
+hasCurrentRange,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,Range,exactly 1,,https://www.theworldavatar.com/kg/ontoems/,has current range
+hasCurrentTrend,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,Trend,,Relationship between a water level measure and its current trend,https://www.theworldavatar.com/kg/ontoems/,has current trend
+hasCurrentTrend,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,Trend,exactly 1,,https://www.theworldavatar.com/kg/ontoems/,has current trend
+CloudCover,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Number,IS-A,,,,Fraction of sky obscured by clouds.,https://www.theworldavatar.com/kg/ontoems/,Cloud cover
+CloudCover,Class,http://purl.org/iot/vocab/m3-lite#CloudCover,EQUIVALENT-TO,,,,Fraction of sky obscured by clouds.,https://www.theworldavatar.com/kg/ontoems/,Cloud cover
+hasObservationLocation,Data Property,,,ReportingStation,http://www.bigdata.com/rdf/geospatial/literals/v1#lat-lon,,Description of observation location (i.e. point location in WGS84),https://www.theworldavatar.com/kg/ontoems/,has observation location
```

Comment:
Please correct me if I am wrong, but the idea is to have this geo-representation only until CEA properly works with the stack and also uses PostGIS for geospatial queries. Hence, I would suggest not to change the ontology explicitly and just instantiate this triple on an instance level. 
Looping in Seb: @sm453 What's your perspective on that?

Reply:
You are right in that the long-term plan is to move all geospatial functionality to PostGIS. Until then, though, practical minimum-effort solutions are acceptable. Feel free to decide if this makes your life easier in the short term, but please do consider how much work it will be to migrate to PostGIS when the time comes...

Reply:
Decided that the ontology won't be extended explicitly, and will remove the added OntoEMSExtension.csv and OntoEMSExtension.owl.

Reply:
Looks like this pull request is empty, then. If nothing further is expected, here, then you can simply close it.